runtime/include/mthdecls.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#if !defined(_WIN64) && !defined(HOST_MINGW)
+#define FLOAT_COMPLEX_TYPE complex float
+#define FLOAT_COMPLEX_CREATE(real, imag) (real + imag * I)
+#define FLOAT_COMPLEX_MUL_CC(a, b) a * b
+#define FLOAT_COMPLEX_ADD_CC(a, b) a + b
+#define FLOAT_COMPLEX_EQ_CC(a, b) a == b
+#define DOUBLE_COMPLEX_TYPE complex double
```

Comment:
This should be defined to `double complex`.

Reply:
```suggestion
#define DOUBLE_COMPLEX_TYPE double complex
```